app/helpers/api/bapi/getCatalogueItems.js

```diff
@@ -28,6 +28,7 @@ export const getCatalogueItems = async ({ supplierId, catalogueItemType }) => {
 
   logger.info(`Retrieving ${catalogueItemType} catalogue items from BAPI for supplier ${supplierId}`);
   const catalogueItems = await getData({ endpoint, logger });
+  catalogueItems.sort((a, b) => ((a.name > b.name) ? 1 : -1));
```

Comment:
Move this to a little helper function within this file. Also can you lowerCase the name for comparing. Otherwise I think we might get something like this;

A Solution
B Solution 
a solution
b solution

Example of where we have done something similar
```
const sortItems = (items, propToSort) => (
  items.sort((itemA, itemB) => {
    const itemAPropValue = itemA[propToSort].toLowerCase();
    const itemBPropValue = itemB[propToSort].toLowerCase();

    if (itemAPropValue < itemBPropValue) return -1;
    if (itemAPropValue > itemBPropValue) return 1;
    return 0;
  })
);
```